sale_manual_delivery/models/sale_order.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Denis Leemann, Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models, api
+from odoo.exceptions import UserError
+from odoo.tools.translate import _
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    manual_delivery = fields.Boolean(
+        string='Manual Delivery',
+        default=False,
+        help="If Manual, the deliveries are not created at SO confirmation.\
+        You need to use the Create Delivery button in order to reserve and \
+        ship the goods."
+    )
+
+    @api.onchange('team_id')
+    def _onchange_team_id(self):
+        self.manual_delivery = self.team_id.manual_delivery \
```

Comment:
because the `onchange` is `api.multi` by default:
```suggestion
        for so in self:
            if so.team_id:
                so.manual_delivery = so.team_id.manual_delivery
            else:
                so.manual_delivery = False
```